usaspending_api/api_docs/api_documentation/recipient/recipient_profile.md

```diff
@@ -0,0 +1,109 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Recipient Profile
+
+These endpoints are used to power USAspending.gov's recipient profile pages. This data can be used to visualize the government spending that pertains to a specific recipient.
+
+# Group Profile Page
+
+These endpoints support the individual Recipient Profile pages that display data for a specific DUNS.
+
+## Recipient Overview [/api/v2/recipient/duns/{recipient_id}/{?year}]
+
+This endpoint returns a high-level overview of a specific recipient, given its id.
+
++ Parameters
+    + `recipient_id`: `0036a0cb-0d88-2db3-59e0-0f9af8ffef57-C` (required, string)
```

Comment:
can we make an explicit note for adding `-C`, `-P`, or `-R` for the recipient level?

Reply:
can we also add a sample request for each endpoint added here?

Reply:
Added both

Reply:
While it is good to include documentation on the expected data, the client should never need to create the ID for searches. The API will provide the correct `recipient_id` and the client will pass it back when appropriate.